### DISCUSSION TITLE
ActionDispatch::HostAuthorization::DefaultResponseApp

### DISCUSSION THREAD

**Discussion by MConsuela** [mastodon/mastodon#29811] [Q&A]
Hello, im trying to find a Error with Image Display, and i get 

"ActionDispatch::HostAuthorization::DefaultResponseApp <MY OWN IP>"

a lot, is there any reason for thos or any way i can unblock this?

![gjPbOXh](https://github.com/mastodon/mastodon/assets/151952217/ace6f0cf-1246-4b44-a4b5-ab584fbb69b7)

Thanks.

**Comment by mjankowski** [comment]
I realize this is old, but do you still have this issue? What's probably happening is some URLs area being requested that do route to your server, but are not configured as valid hostnames. Reasons for that could vary, would need more details on setup.
